docs/operational_guide/topology.md

```diff
@@ -0,0 +1,173 @@
+# Topology
+
+## Overview
+
+M3DB stores its topology (mapping of which hosts are responsible for which shards) in [etcd](https://coreos.com/etcd/). There are three possible states that each host/shard pair can be in:
```

Comment:
Maybe also say we call topology a `Placement` here?

Reply:
also, it'd be good to explain the relationship between cluster/namespace/shard & topology here.

i.e. 
- A cluster has exactly 1 Topology/Placement. The Topology/Placement maps Cluster shard replicas to hosts. 
- A cluster has 0 or more namespaces. Each host serves every namespace for the shards it owns

Reply:
done